$OpenBSD: patch-src_pooler_app_src,v 1.1.1.1 2013/03/08 10:55:00 jasper Exp $

Set the version, so rebar doesn't need to invoke git to figure it out.

--- src/pooler.app.src.orig	Sun Feb 24 22:54:38 2013
+++ src/pooler.app.src	Sun Feb 24 22:54:47 2013
@@ -1,7 +1,7 @@
 {application, pooler,
  [
   {description, ""},
-  {vsn, git},
+  {vsn, "${VERSION}"},
   {registered, []},
   {applications, [
                   kernel,
